### PR TITLE
9.0 mrp brewing fix raw materials

### PR DESCRIPTION
**Pull request by vvrossem**
Raw Material reports include :
- date display according to user timezone
- outputs for customers in addition to production and inventory

### CODE REVIEW

File: mrp_brewing/report/report_stock_raw_materials.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
                                         <span t-att-style="style" t-esc="a.quantity_after_move" />
                                     </td>
                                 </t>
-                                <t t-if="a.location_dest_id.usage in ['production','inventory']">
+                                <t t-if="a.location_dest_id.usage in ['production','inventory','customer']">
                                     <td/>
```
Comment (houssine78) [inline mrp_brewing/report/report_stock_raw_materials.xml:69]: I don't understand why location of usage customer appear in this report. This is the raw material report. So the report telling what are the stock variation for the raw material product. I don't think we should add customer usage here

**Review by houssine78 — CHANGES_REQUESTED** [review]
could you please take my comment into account

**Comment by vvrossem** [comment]
@houssine78 ok to merge ? (cfr. email 17.10.19 - 15:13)

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
@vvrossem ok pour moi !

**Pull request merged**